### PR TITLE
fix(hydra): remove call to awscli to download keys

### PR DESCRIPTION
**Pull request by fruch**
this would complicate the sisuation once we need
want to use diffent aws profiles, so this move into `sct.py` itself, where we'll be able to even do that selectivly  as needed, or push is down further to places it's gonna be used (i.e. no every command need
validating/downloading those)

### Testing

- [ ]

### PR pre-checks (self review)

- [x] I added the relevant `backport` labels
- [x] I didn't leave commented-out/debugging code

### Reminders

- Add New configuration option and document them (in `sdcm/sct_config.py`)
- Add unit tests to cover my changes (under `unit-test/` folder)
- Update the Readme/doc folder relevent to this change (if needed)

### CODE REVIEW

File: sct.py [hunk 1]
```diff
@@ -185,6 +186,10 @@ def getTestCaseNames(self, testCaseClass):
               help="Install paths for extra python packages to install, scylla-cluster-plugins for example")
 def cli():
     LOGGER.info("install-bash-completion current path: %s", os.getcwd())
+    if not is_using_aws_mock():
+        key_store = KeyStore()
+        key_store.sync(keys=['scylla-qa-ec2', 'scylla-test', 'scylla_test_id_ed25519'],
+                       local_path=Path('~/.ssh/').expanduser(), permissions=0o0600)
```
Comment (roydahan) [inline sct.py:192]: What is this key "scylla_test_id_ed25519"?

**Review by roydahan — DISMISSED** [review]

File: sct.py (hunk 1 above)
Comment (fruch) [reply sct.py:192]: In rocky9 we started using this key, we should slowly move everything to this type of key, and remove all the tricks we do in ssh servers to enable old types

**Comment by nyh** [comment]
See also https://github.com/scylladb/scylla-cluster-tests/issues/4782

File: sct.py (hunk 1 above)
Comment (nyh) [reply sct.py:192]: It's an interesting question. I don't think there's a clear consensus of why the newer elliptic-key algorithm is better than the older RSA. Neither are "quantum safe", or offer any other huge advantage over the other. There are some small advantages like key length which doesn't matter to us at all. I think all the hoopla against RSA comes from this - https://www.openssh.com/txt/release-8.2 - it isn't RSA which sucks but SHA-1, and the original openssh RSA implementation used SHA-1 (the current one uses SHA-2 by default, and doesn't suffer from this problem).

**Review by vponomaryov — APPROVED** [review]
LGTM

**Review by soyacz — APPROVED** [review]
LGTM

File: sct.py (hunk 1 above)
Comment (fruch) [reply sct.py:192]: I care more about things that would work best out of the box, then which key is really superior.

And the rsa ones don't work out of the box, and were causing use issues cause we need to do all kinds of work arounds to enable it

**Pull request merged**